src/main/java/org/fundacionjala/coding/erik/SortInner.java

```diff
@@ -0,0 +1,34 @@
+package org.fundacionjala.coding.erik;
+
+import java.util.Arrays;
+
+/**
+ * this is class for sort inner.
+ */
+public class SortInner {
+    private static final int LENGTH = 3;
+    private static final int NUMBER = 1;
+    private static final int NUMBER_CIRO = 0;
+
+    /**
+     * this class is to order the string.
+     *
+     * @param words to evaluate.
+     * @return the string content in descending order.
+     */
+
+    public String sortTheInnerContent(final String words) {
+        String[] word = words.split(" ");
+
+        for (int i = 0; i < word.length; i++) {
+            if (word[i].length() > LENGTH) {
+                char[] contentWord = word[i].substring(NUMBER, word[i].length() - NUMBER).toCharArray();
+                Arrays.sort(contentWord);
+                String ini = word[i].substring(NUMBER_CIRO, NUMBER);
+                String fin = word[i].substring(word[i].length() - NUMBER);
+                word[i] = ini.concat(new StringBuilder(new String(contentWord)).reverse().toString().concat(fin));
+            }
+        }
+        return String.join(" ", word);
```

Comment:
This may be refactored like this.
```java
        String[] wordArray = words.split(" ");
        for (int i = 0; i < wordArray.length; i++) {
            if (wordArray[i].length() > LENGTH) {
                String[] contentWord = wordArray[i].substring(NUMBER, wordArray[i].length() - NUMBER).split("");
                Arrays.sort(contentWord, Collections.reverseOrder());
                wordArray[i] = wordArray[i].substring(NUMBER_CIRO, NUMBER)
                        .concat(String.join("", contentWord))
                        .concat(wordArray[i].substring(wordArray[i].length() - NUMBER));
            }
        }
        return String.join(" ", wordArray);
```